packages/core/src/components/RadioButton/RadioButtonGroup.tsx

```diff
@@ -60,7 +61,7 @@ const RadioButtonGroup: React.FC<RadioButtonGroupProps> = ({
     <View style={[{ minHeight: 40 }, style]} {...rest}>
       <Provider
         value={{
-          value: internalValue || "",
+          value: internalValue ?? "",
```

Comment:
any reason you're changing this to `??`?

Reply:
@peter I like it.  But it is changed back because you asked.